src/aleph_client/asynchronous.py

```diff
@@ -572,6 +573,8 @@ async def get_messages(
     end_date: Optional[Union[datetime, float]] = None,
     session: Optional[ClientSession] = None,
     api_server: str = settings.API_HOST,
+    ignore_invalid_messages: bool = True,
+    invalid_messages_log_level: Optional[int] = None,
```

Comment:
There's 'logging.NOT_SET' which is equal to 0. Using this would avoid the need for 'if invalid_messages_log_level' checks.

Reply:
Using the following then ?
```suggestion
    invalid_messages_log_level: int = logging.NOT_SET,
```